### DISCUSSION TITLE
OpenMage Feed Notifications in Backend

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1655] [Ideas]
OpenMage is using this URL for feeding notifications in Backend:

```
<feed_url>www.openmage.org/Web_Notifications/notifications_v1.rss</feed_url>
```

If you check this page content you will see an item from 2019, "Thank you for installing OpenMage M1 LTS!".

I suggest to use this feature and send periodically notifications related to OpenMage, at least for new releases and features. For example, there is a new feature related to ordering Terms & Conditions in checkout, we improved the speed for X, we fixed the Y.

**Comment by fballiano** [comment]
I don't know, I never liked that feature, I just hope it would be removed, we don't need any external connection in our store :-)

**Reply by luigifab** [reply]
Yes, remove it.
Admin users of OpenMage are not interested.
And "dev admin users" doesn't want click mark as read, on: local 1, local 2, dev, test, staging, prod environments.

If I can choose, I prefer to subscribe to a newsletter.

**Comment by addison74** [comment]
I propose that the feed from the openmage.org website be updated with information when a new version is released. This way the administrators will immediately get a notification in Inbox. The procedure is not complicated and takes only a few minutes.

Another advantage would be that there are many OpenMage-based stores whose administrators are unaware that this project is still active.

**Reply by justinbeaty** [reply]
I would be okay with new versions announced only, but I do not want notifications for various features.
